src/App.vue

```diff
@@ -3,6 +3,8 @@ import { RouterLink, RouterView } from 'vue-router'
 </script>
 
 <template>
+  <Sidebar /> <!-- Add Sidebar To Page -->
```

Comment:
This looks sick! Also, you can remove the existing navigation links that were there since this replaces them.